extension/popup/popupView.jsx

```diff
@@ -286,10 +286,6 @@ this.popupView = (function() {
             ></path>
           </svg>
         </div>
-        <div id="moz-voice-privacy">
-          <strong>For Mozilla internal use only</strong>
-          {/* <a href="">How Mozilla ensures voice privacy</a> */}
```

Comment:
@awallin do we want to restore that privacy link? I forgot that I overwrote it with the internal use message.

Reply:
Because our FAQ needs work let's leave the link out for now. I'll open a new issue for the FAQ and when that is in better shape we'll add the link.